src/Faker/Provider/es_CL/Person.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Faker\Provider\es_Cl;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}} {{suffix}}',
+        '{{firstNameMale}} {{lastName}} {{suffix}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}} {{suffix}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}} {{suffix}}',
+    );
+
+    /**
+     * 
+     * {@link} https://www.registrocivil.cl/Servicios/Estadisticas/Archivos/NombresComunes/damas.html
+     * {@link} https://www.registrocivil.cl/Servicios/Estadisticas/Archivos/NombresComunes/varones.html
+     */
+    protected static $firstNameMale = array(
+         'Benjamin', 'Vicente', 'Martin', 'Matias', 'Joaquin', 'Agustin', 'Cristobal', 'Maximiliano', 'Sebastian', 'Tomas', 'Diego', 'Jose', 'Nicolas', 'Felipe', 'Lucas', 'Alonso', 'Bastian', 'Juan', 'Gabriel', 'Ignacio', 'Francisco', 'Renato', 'Maximo', 'Mateo', 'Javier', 'Daniel', 'Luis', 'Gaspar', 'Angel', 'Fernando', 'Carlos', 'Emilio', 'Franco', 'Cristian', 'Pablo', 'Santiago', 'Esteban', 'David', 'Damian', 'Jorge', 'Camilo', 'Alexander', 'Rodrigo', 'Amaro', 'Luciano', 'Bruno', 'Alexis', 'Victor', 'Thomas', 'Julian'
```

Comment:
please split the line by first letter for better readability.

Same for other lists.